File: JAVA BASICS/fibonacci.java
import java.util.*;

public class fibonacci {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter number of terms: ");
        int x = sc.nextInt();

        int t1 = 0, t2 = 1;

        if (x <= 0) {
            System.out.println("Please enter a positive integer.");
        } else if (x == 1) {
            System.out.println("Fibonacci Series: " + t1);
        } else {
            System.out.print("Fibonacci Series: " + t1 + ", " + t2);

            for (int i = 3; i <= x; i++) {
                int nextTerm = t1 + t2;
                System.out.print(", " + nextTerm);
                t1 = t2;
                t2 = nextTerm;
            }
            System.out.println();  
        }
    }
}

File: BASICS/fibonacci.cpp
#include<iostream>
using namespace std;
int main(){
    int t1 =0, t2=1,nextTerm =0, n;
    cout<<"enter a no of terms";
    cin>>n;
    cout<<"Fibonacci series :"<< t1 <<"," <<t2;
    for(int i=3;i<=n;i++){
        nextTerm = t1+t2;
        cout<<','<< nextTerm;
        t1 =t2;
        t2 = nextTerm;
    }
    cout<<endl;
}